lib/access_import/volunteer_transform.rb

```diff
@@ -0,0 +1,32 @@
+require 'acc_utils'
+
+class VolunteerTransform
+  include AccUtils
+
+  def initialize(*accessors)
+    accessors.each do |accessor|
+      instance_variable_set("@#{accessor.class.name.underscore}", accessor)
+    end
+  end
+
+  # could be needed relations
```

Comment:
```ruby
k_traeger = @kosten_traeger.find(personen_rolle[:fk_Kostenträger])
```
Kostenträger are the authorities responsible for a client, maybe also the accompainment. Thats to be reverse engeneered.

```ruby
journals = @journale.where_haupt_person(haupt_person[:pk_Hauptperson])
```

Journals are pretty similar to our journals.

```ruby
einsaetze = @fw_einsaetze.where_personen_rolle(personen_rolle[:pk_PersonenRolle])
```

Could possibly play part in resolving accompainments, but not yet sure

```ruby
entschaedigungen = @fw_entschaedigung.where_personen_rolle(personen_rolle[:pk_PersonenRolle])
```

These are Spesen. Espenses pay back.

```ruby
konto_angaben = @konto_angaben.where_haupt_person(haupt_person[:pk_Hauptperson])
```
Volunteers account informations

```ruby
stunden_erfassungen = @stundenerfassung.where_personen_rolle(personen_rolle[:pk_PersonenRolle])
```

A Feature we do not have yet: Volunteers recording their hours.